### DISCUSSION TITLE
[Esame 23 Giugno 2020] Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#241] [Esami - Bertini @ Informatica]
![immagine](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/909dab59-e7c9-49ab-99f4-7a7cfc6cc8df)

**Comment by LeoCosta02** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/128373356/6fc63b23-570c-4e3f-9c92-29345905a3ba)

**Comment by LeoCosta02** [comment]
il secondo punto non saprei come svolgerlo visto che le distanze dal valore atteso non sono simmetrici, sul libro ho visto esempi dove le distanze dal valore atteso sono solo simmetrici

**Comment by Elia-Belli** [comment]
### Disuguaglianza di Chebyshev
Sia $X$ una v.a. arbitraria e $\lambda\gt 0$ allora abbiamo che:

$$\mathbb{P}(|X-\mathbb{E}(X)|\geq \lambda)\leq \frac{1}{\lambda^2}\cdot \mathbb{V}(X)$$

---

**(1)** Rimuoviamo il modulo perchè ci interessano solo valori positivi di $X$ e troviamo $\lambda$:

$$\mathbb{P}(X\geq 25+\lambda)\leq \frac{1}{\lambda^2}\cdot 4$$

$$25+\lambda =35\to \lambda =10\ \ \ \ \Rightarrow \ \ \ \ \mathbb{P}(X\geq 35)\leq \frac{1}{10^2}\cdot 4=0.04$$

> La disuguaglianza di Chebyshev ci permette di calcolare un limite superiore per la probabilità desiderata

**(2)** In questo caso ci conviene riscrivere la probabilità da stimare in un'altra forma:

$$\mathbb{P}(21\leq X\leq 30)=1-[\mathbb{P}(X\lt 21) + \mathbb{P}(X\gt 30)]=\mathbb{P}(X\geq 21)-\mathbb{P}(X\gt 30)$$

$$\mathbb{P}(X\geq 25-4)\leq \frac{4}{(-4)^2}=\frac{1}{4}$$

$$\mathbb{P}(X\geq 25+5)\leq \frac{4}{5^2}=\frac{4}{25}$$

$$\mathbb{P}(21\leq X\leq 30)\leq \frac{1}{4}-\frac{4}{25}=\frac{9}{100}=0.09$$

**Reply by UnoCasuale** [reply]
Come ipotesi del teorema si ha che λ >0 ma nel caso P(x>25-4)  λ=-4 quindi il teorema non é applicabile, come é stato risolto quel passaggio?
